client/components/Insurance-Card/InsurancePage.vue

```diff
@@ -4,10 +4,13 @@
 <template>
   <main>
     <section>
-      <!-- <button>
-          Add contact
-        </button> -->
-      <CreateInsuranceForm />
+      <button v-if="!showForm" class="btn-primary" @click="showForm=true">
+        Add insurance
+      </button>
+      <button v-if="showForm" class="btn-primary" @click="showForm=false">
+        Add insurance
+      </button>
```

Comment:
Maybe this could be simplified to one component by having a toggling method for `showForm`? Something like `@click="toggleShowForm"`

Reply:
I don't think the change would significantly simplify the code – it means adding a toggle method and essentially keeping similar logic as the current one